pubspec.lock

```diff
@@ -1123,5 +1123,5 @@ packages:
     source: hosted
     version: "1.0.0"
 sdks:
-  dart: ">=3.1.0 <4.0.0"
+  dart: ">=3.2.0-194.0.dev <4.0.0"
```

Comment:
Appears you are using either the beta or the main branch, may you switch to the stable branch of flutter